types/ioredis/index.d.ts

```diff
@@ -673,7 +677,7 @@ declare namespace IORedis {
         hget(key: KeyType, field: string, callback?: (err: Error, res: string | string) => void): Pipeline;
         hgetBuffer(key: KeyType, field: string, callback?: (err: Error, res: Buffer) => void): Pipeline;
 
-        hmset(key: KeyType, field: string, value: any, ...args: string[]): Pipeline;
+        hmset(key: KeyType, ...args: string[]): Pipeline;
```

Comment:
Why change this?

Reply:
same as https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33190#discussion_r258020914